pkg/apiserver/internal/util/util.go

```diff
@@ -28,3 +32,95 @@ func TimestampProto(t *metav1.Time) (*timestamp.Timestamp, error) {
 	}
 	return ptypes.TimestampProto(t.Time)
 }
+
+func FromMetav1(obj metav1.ObjectMeta) (*v1.ObjectMeta, error) {
```

Comment:
This function is not used anywhere, and also I think these helper functions should be versioned under `/v1` folder?
If this function is not used in this PR, please remove it, I have added a similar helper function in the refactor PR to share between all our services.

Reply:
done